drake/examples/KukaArm/urdf/iiwa14.urdf

```diff
@@ -0,0 +1,435 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from iiwa14.urdf.xacro              | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
+<robot name="iiwa14" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <!-- Import Rviz colors -->
+  <material name="Black">
+    <color rgba="0.0 0.0 0.0 1.0"/>
+  </material>
+  <material name="Blue">
+    <color rgba="0.0 0.0 0.8 1.0"/>
+  </material>
+  <material name="Green">
+    <color rgba="0.0 0.8 0.0 1.0"/>
+  </material>
+  <material name="Grey">
+    <color rgba="0.4 0.4 0.4 1.0"/>
+  </material>
+  <material name="Orange">
+    <color rgba="1.0 0.423529411765 0.0392156862745 1.0"/>
+  </material>
+  <material name="Brown">
+    <color rgba="0.870588235294 0.811764705882 0.764705882353 1.0"/>
+  </material>
+  <material name="Red">
+    <color rgba="0.8 0.0 0.0 1.0"/>
+  </material>
+  <material name="White">
+    <color rgba="1.0 1.0 1.0 1.0"/>
+  </material>
+  <!--Import the lbr iiwa macro -->
+  <!-- Import all Gazebo-customization elements, including Gazebo colors -->
+  <!-- Import Transmissions -->
+  <!-- Include Utilities -->
+  <!-- The following macros are adapted from the LWR 4 definitions of the RCPRG - https://github.com/RCPRG-ros-pkg/lwr_robot -->
+  <!--Little helper macros to define the inertia matrix needed for links.-->
+  <!--Cuboid-->
+  <!--Cylinder: length is along the y-axis! -->
+  <!-- some constants -->
+  <!-- Fix to world just for testing -->
+  <link name="world"/>
```

Comment:
Yes, I think we should modify Drake's URDF parser to allow for a world definition. It will basically override the `weld_to_frame` parameter. Better yet, we should probably throw an exception if both the world frame is specified via URDF and the `weld_to_frame` is specified as a method call parameter. I will create an issue now.